File: Web/ClientApp/src/app/models/et/evaluationMaster.ts
import { EntityBase } from "../entityBase";
import { EvaluationMasterDetail } from "./evaluationMasterDetail";

export class EvaluationMaster extends EntityBase {
    id: string;
    positionCode: string;
    active?: boolean;
    evaluationMasterDetails: EvaluationMasterDetail[];

    constructor() {
        super();
        this.evaluationMasterDetails = [] as EvaluationMasterDetail[]
    }
}

File: Domain/Entities/ET/EvaluationMaster.cs
using System;
using System.Collections;
using System.Collections.Generic;

namespace Domain.Entities.ET;

public class EvaluationMaster : EntityBase
{
    public Guid Id { get; set; } = System.Guid.NewGuid();
    public string PositionCode { get; set; }
    public bool Active { get; set; } = true;
    public ICollection<EvaluationMasterDetail> EvaluationMasterDetails { get; set; } = new List<EvaluationMasterDetail>();
}
